__tests__/app.test.js

```diff
@@ -290,6 +290,54 @@ describe('GET api/articles/:article_id/comments', () => {
     });
 });
 
-describe('Name of the group', () => {
-    
-});
+describe('POST /api/articles/:article_id/comments', () => {
+    test('status: 200, returns with new inserted comment', () => {
+        const id =2;
+        const newComment = {
+            author: "rogersop",
+            body: "This article absolutely smacks dude",
+        }
+        return request(app)
+        .post(`/api/articles/${id}/comments`)
+        .expect(201)
+        .send(newComment)
+        .then(({body})=>{
+            // console.log(body)
+            // expect(body.comments).toEqual({ 
+            //     comment_id: 19, 
+            //     ...newComment,
+            //     article_id: 2,
+            //     votes: 0,
+            //     created_at: "2022-10-13T10:18:00.649Z"
+            // })
+            expect(body.comments).toEqual(
+                expect.objectContaining({
+                  author: "rogersop",
+                  body: "This article absolutely smacks dude",
+                })
+              );
+        }) 
+    });
+  
+    test('status:404, correct data type but id does not exist to update ', () => {
+        const id = 287;
+        return request(app)
+        .post(`/api/articles/${id}/comments`)
+        .send({author:"rogersop", body: "This article absolutely smacks dude",})
+        .expect(404)
+        .then(({body})=>{
+            expect(body.msg).toBe("This id is not found")
+        })
+    })
+    test('status: 400, invalid Id not a number cant patch wrong data type', () => {
+        const id = "random"
+        return request(app)
+        .post(`/api/articles/${id}/comments`)
+        .send({author: "rogersop", body: "This article absolutely smacks dude",})
+        .expect(400)
+        .then(({body}) =>{
+            expect(body.msg).toBe("ID/vote is invalid")
+        })
+        
+    });
+    });
```

Comment:
Some sad paths to consider
  - [ ] Status 400, missing required field(s), e.g. no author or body properties
  - [ ] Status 404, author(username) does not exist
  - [ ] Status 201, ignores unnecessary properties 
         newComment = { 
                                      author: "rogersop",
                                      body: "Testing",
                                      whatever: "something should be ignored"
                                      }